src/CMakeLists.txt

```diff
@@ -42,21 +43,13 @@ if (NOT ${PLATFORM} MATCHES "Web")
     include(GlfwImport)
 endif ()
 
+add_library(raylib ${raylib_sources} ${raylib_public_headers})
+
 # Sets additional platform options and link libraries for each platform
 # also selects the proper graphics API and version for that platform
 # Produces a variable LIBS_PRIVATE that will be used later
 include(LibraryConfigurations)
 
-if (USE_AUDIO)
```

Comment:
Why was this hunk deleted?
I see:
`USE_AUDIO is redundant in the presence of the already existent and more descriptive SUPPORT_MODULE_RAUDIO.`
Which is okay reasoning... but the option still shows up in ccmake or cmake-gui, and presumably does nothing.
It could maybe break existing users that rely on this (though I doubt there are many...)
I would be happy to review this in a separate PR, but I don't think it belongs in this one.

Reply:
@lzralbu this would be more convenient to discuss elsewhere, like the raylib discord server. Ask there and I can help more directly.
I personally do this with `git rebase -i`.
You can change `pick` to `drop` in the editor for the commits you would like to remove.
After you've done this, you have to force push `git push --force`.
There's some more info available here:
https://git-rebase.io